src/audio/sound_effects.rs

```diff
@@ -61,6 +61,8 @@ struct PlaySoundEffect {
 
 impl Command for PlaySoundEffect {
     fn apply(self, world: &mut World) {
+        // If you need more complex behavior, use `world.run_system_once_with`,
+        // as demonstrated for `PlaySoundtrack`.
```

Comment:
```suggestion
        // as demonstrated with `PlaySoundtrack`.
```
I'm leaning that this reads slightly better, but not 100%. Can ignore this.